azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/fragment/calling/lobby/LobbyOverlayViewModel.kt

```diff
@@ -3,15 +3,20 @@
 
 package com.azure.android.communication.ui.presentation.fragment.calling.lobby
 
+import com.azure.android.communication.ui.configuration.LocalizationProvider
 import com.azure.android.communication.ui.redux.state.CallingStatus
 import kotlinx.coroutines.flow.MutableStateFlow
 import kotlinx.coroutines.flow.StateFlow
 
-internal class LobbyOverlayViewModel {
+internal class LobbyOverlayViewModel(private val localizationProvider: LocalizationProvider) {
     private lateinit var displayLobbyOverlayFlow: MutableStateFlow<Boolean>
 
     fun getDisplayLobbyOverlayFlow(): StateFlow<Boolean> = displayLobbyOverlayFlow
 
+    fun getLocalizationProvider(): LocalizationProvider {
```

Comment:
should be after init {} also consider getter syntax.

Reply:
Conflict resolve made it look like its this PR's changes. But I'm noting down this point and will bring up in the next refactoring meeting.

Reply:
Current refactoring dismisses getLocalizationProvider method so I think we are good.